docs/skinning.md

```diff
@@ -87,31 +87,38 @@ Variable name | Default | Description
  `--alpheios-desktop-popup-icon-color-hover` | `--alpheios-icon-color-hover` | Popup header icons color - __hover__ state
  `--alpheios-desktop-popup-icon-bg-hover` | `--alpheios-toolbar-bg-color` | Popup header - icon background - __hover__ state
  `--alpheios-desktop-popup-icon-color-active` | `--alpheios-icon-color-active` |  Popup header - icons color - __active__ state
+ `--alpheios-desktop-popup-icon-color-disabled` | `--alpheios-icon-color-disabled` |  Popup header - icons color - __disabled__ state
+ `--alpheios-desktop-popup-icon-color-pressed` | `--alpheios-icon-color-pressed` |  Popup header - icons color - __pressed__ state
+`--alpheios-desktop-popup-icon-bg-disabled` | `--alpheios-toolbar-bg-color` |  Popup header - icons color - __disabled__ state
   | | 
  `--alpheios-desktop-popup-progress-bar-line` | `--alpheios-toolbar-bg-color` | Popup progress bar color
   `--alpheios-desktop-popup-link-color` | `--alpheios-link-color` | Popup link color (grammar links)
  `--alpheios-desktop-popup-link-color-hover` | `--alpheios-link-color-hover` | Popup link color (grammar links) - __hover__ state
- `--alpheios-desktop-popup-credit-link-color` | `--alpheios-link-special-color` | Popup - color for credits link
- `--alpheios-desktop-popup-credit-link-color-hover` | `--alpheios-link-special-color-hover` | Popup - color for credits link - __hover__ state
+ `--alpheios-desktop-popup-credit-link-color` | `#FFC24F` | Popup - color for credits link

```

Comment:
@balmas , I believe that you will earase all my changes to documents with this update.
May be it would be better to update this `skinning.md` first?

Reply:
hmm. I thought those changes were in the alpheios-release-3.0 branch. I will double check and fix this when I merge.